examples/app.ts

```diff
@@ -4,9 +4,9 @@
 /* eslint-disable no-console */
 import fs from 'fs';
 import path from 'path';
-import { Buffer } from 'buffer';
 import { getLocalAlgodClient, getLocalAccounts, compileProgram } from './utils';
 import algosdk from '../src';
+import { base64ToBytes, base64ToString } from '../src/encoding/binarydata';
```

Comment:
Could you remove this import and instead reference the functions like `algosdk.base64ToBytes`? That's how we prefer consumers to import definitions

Reply:
Good catch, changed the functions to prefix with `algosdk`

Reply:
The other example files need to be updated as well